### PR TITLE
[VSEE-1162] Add snyk2spdx Dockerfile

### PR DESCRIPTION
**Pull request by elfotografo007**
# Which other branches do you want a technical writer to cherry-pick this PR to (if any)?
This change targets TAP 1.7.0 and 1.6.x.

It's best to PR to the most recent relevant branch and leave all the cherry-picking to the
writer, except where earlier branches require factual changes to the content.
For more information about the branches, see https://github.com/pivotal/docs-tap#branches

### CODE REVIEW

**Comment by a-peek4** [comment]
Hi @elfotografo007 Please ping me when you are ready to take this out of draft state as the doc freeze is on Nov 3

File: scst-scan/ivs-snyk.hbs.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ For information about the IVS specification, see [Configuration Options](ivs-cre
 apiVersion: app-scanning.apps.tanzu.vmware.com/v1alpha1
 kind: ImageVulnerabilityScan
 metadata:
-  name: snyk-ivs
+  name: snyk-ivs-spdx
```
Comment (ryan-a-baker) [inline scst-scan/ivs-snyk.hbs.md:37]:
```suggestion
  name: snyk-ivs
```

I'd just leave it as Snyk

File: scst-scan/ivs-snyk.hbs.md [hunk 2]
```diff
@@ -77,7 +81,34 @@ Where:
...
+        COPY --from=build /build-dir/dist/ /app/dist/
+        COPY --from=build /build-dir/node_modules /app/node_modules
+        COPY --from=build /build-dir/snyk2spdx-linux /app/bin/snyk2spdx
+
+        ENTRYPOINT ["/app/bin/snyk2spdx"]
+        CMD ["/app/bin/snyk2spdx"]
+        ```
+    3. Build and push the image to a registry. Replace `SNYK2SPDX-IMAGE` with
+      the new image you just built.
+    > **Note** The `snyk2spdx` output is not conformant to the [verification process](./verify-app-scanning-supply-chain.hbs.md) so VMware does not guarantee the results.
```
Comment (ryan-a-baker) [inline scst-scan/ivs-snyk.hbs.md:111]:
```suggestion
    > **Note** The `snyk2spdx` output is not conformant to the [verification process](./verify-app-scanning-supply-chain.hbs.md) although the results may be ingested to the Tanzu Application Platform metadata store, VMware does not guarantee the accuracy of the results.
```

**Comment by elfotografo007** [comment]
@a-peek4, the PR is ready for review.

**Pull request merged**